reinstall-cli/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "bootc-reinstall"
+version = "0.1.9"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/containers/bootc"
+readme = "README.md"
+publish = false
+# For now don't bump this above what is currently shipped in RHEL9.
+rust-version = "1.75.0"
+
+# See https://github.com/coreos/cargo-vendor-filterer
+[package.metadata.vendor-filter]
+# This list of platforms is not intended to be exclusive, feel free
+# to extend it.  But missing a platform will only matter for the case where
+# a dependent crate is *only* built on that platform.
+platforms = ["x86_64-unknown-linux-gnu", "aarch64-unknown-linux-gnu", "powerpc64le-unknown-linux-gnu", "s390x-unknown-linux-gnu", "riscv64gc-unknown-linux-gnu"]
+
+[dependencies]
+anyhow = { workspace = true }
+bootc-lib = { version = "1.0", path = "../lib" }
```

Comment:
I'm fine with this short term but in the medium term...it seems like it'd be cleaner/clearer to have this use only at most our `utils` crate and is otherwise just a consumer of existing public APIs which I think is pretty much the case, it's just you refactored the tracing bits to be shared.

But I think we could actually put the tracing bits into `utils` maybe? Dunno.

Reply:
Yeah I wanted to have it in a separate crate but didn't want to create one

Didn't know we had a utils crate

Anyway it seems the resulting RPM is still very tiny. The linker must discard all the entirety of the unused bootc library

Reply:
> Didn't know we had a utils crate

It may not even be needed here, it doesn't have that much.

> Anyway it seems the resulting RPM is still very tiny. The linker must be discarding the entirety of the bootc library save for the used bit

LTO can work really well, yeah.

I'm not worried about size really, it's just more that ideally we show that one can build *external* installer tools on top of our publicly exposed APIs.

Reply:
Can't say I fully understand your intentions and remarks with regards to exposed APIs though. The new binary is not really a consumer of bootc functionality, it's just a command runner. The only reason I made it use the library in the first place was just to share a bit of tracing boilerplate

Reply:
Yep we're agreed!